packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```diff
@@ -173,9 +173,646 @@
+        "type": "object",
+        "required": [
+          "accountId",
+          "quorum"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "quorum": {
+            "type": "number"
+          }
+        }
+      },
+      "AddAssetQuantityRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Add Asset Quantity.",
+        "type": "object",
+        "required": [
+          "assetId",
+          "amount"
+        ],
+        "properties": {
+          "assetId": {
+            "type": "string"
+          },
+          "amount": {
+            "type": "number"
+          }
+        }
+      },
+      "SubtractAssetQuantityRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Subtract Asset Quantity.",
+        "type": "object",
+        "required": [
+          "assetId",
+          "amount"
+        ],
+        "properties": {
+          "assetId": {
+            "type": "string"
+          },
+          "amount": {
+            "type": "number"
+          }
+        }
+      },
+      "TransferAssetRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Transfer Asset.",
+        "type": "object",
+        "required": [
+          "srcAccountId",
+          "destAccountId",
+          "assetId",
+          "description",
+          "amount"
+        ],
+        "properties": {
+          "srcAccountId": {
+            "type": "string"
+          },
+          "destAccountId": {
+            "type": "string"
+          },
+          "assetId": {
+            "type": "string"
+          },
+          "description": {
+            "type": "string"
+          },
+          "amount": {
+            "type": "number"
+          }
+        }
+      },
+      "GetSignatoriesRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Signatories.",
+        "type": "object",
+        "required": [
+          "accountId"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          }
+        }
+      },
+      "GetAccountRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account.",
+        "type": "object",
+        "required": [
+          "accountId"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          }
+        }
+      },
+      "GetAccountDetailRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account Detail.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "key",
+          "writer",
+          "pageSize",
+          "paginationKey",
+          "paginationWriter"
```

Comment:
The command https://iroha.readthedocs.io/en/develop/develop/api/queries.html#get-account-detail has few params optional